contracts/src/v0.8/dev/ArbitrumValidator.sol

```diff
@@ -302,6 +295,8 @@ contract ArbitrumValidator is TypeAndVersionInterface, AggregatorValidatorInterf
       return true;
     }
 
+    // Excess gas on L2 will be sent to the L2 xDomain alias address of this contract
+    address refundAddr = AddressAliasHelper.applyL1ToL2Alias(address(this));
```

Comment:
At the end we are doing what `createRetryableTicket` does for us. If refundAddr will always be this contract aliased, we can let Arbitrum do the aliasing for us

Reply:
That's true, but we'd need to introduce some trust as they have a `shouldRewriteSender` toggle available: https://github.com/OffchainLabs/arbitrum/blob/38e1600c30ef64cba8e52dbafe25af80f2591d38/packages/arb-bridge-eth/contracts/bridge/Inbox.sol#L337

That's the reason I've decided to take control of the aliasing.

Reply:
Would be nice to have some way to inspect this address now that is not on GasConfig. As it's a constant now, could we do the same as other constants and make it `address immutable public`?